File: python/command_line.py
#!/usr/bin/env python

# Usage: ./command_line.py [arg1] [arg2] ...

def command_line(arg):
    # TODO: implementation
    print arg

if __name__ == "__main__":
    import sys
    # pylint: disable=E1120
    command_line(*sys.argv[1:])

File: java/src/com/rwu/templates/CommandLine.java
package com.rwu.templates;

public class CommandLine {

    /**
     * Usage: java -jar command-line.jar [arg]
     */
    public static void main(String[] args) {
        CommandLine cl = new CommandLine();
        cl.commandLine(args);
    }

    public void commandLine(String... args) {
        // TODO: implementation
        if (args.length > 0) {
            System.out.println("Running function with arg: " + args[0]);
        } else {
            System.out.println("Running function with arg: " + null);
        }
    }
}
